Tools/AP_Periph/AP_Periph.h

```diff
@@ -371,6 +371,11 @@ class AP_Periph_FW {
         bool locked;
     } uart_monitor;
 #endif
+
+#if CONFIG_HAL_BOARD == HAL_BOARD_SITL
+    SITL::SIM sitl;
+    AP_AHRS ahrs;
```

Comment:
```suggestion
#endif
#if AP_AHRS_ENABLED
        AP_AHRS ahrs;
```

... and suitable guards.

Reply:
done